new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/Configuration.php

```diff
@@ -44,7 +44,7 @@ private function defineVariablesFromContants($constants, $prefix)
     {
         foreach ($constants as $key => $value) {
             $var = $this->removePrefix($key, $prefix);
-            $var = $this->screamingCaseToCamelCase($var);
+            $var = CaseConverter::screamingToCamel($var);
```

Comment:
Please add a `use` statement for CaseConverter at the beginning of the file.

Reply:
They are in the same namespace, shouldn't it be fine?

Reply:
I like it explicit.

Reply:
How explicit are we talking?

```php
use CaseConverter;
```
or
```php
use RobinTheHood\ModifiedStdModule\Classes\CaseConverter;
```

Reply:
2.